core/src/main/java/org/verapdf/policy/SchematronOperation.java

```diff
@@ -0,0 +1,180 @@
+package org.verapdf.policy;
+
+import org.verapdf.features.objects.Feature;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Enum with operations on elements of Feature.FeatureType that can be used in
+ * policy configuration file.
+ *
+ * @author Sergey Shemyakov
+ */
+public enum SchematronOperation {
+
+	// Any type
+	PRESENT(false, "is present"),
+	NOT_PRESENT(false, "is not present"),
+
+	//Numbers and strings
+	IS_EQUAL(true, "equals to"),
+	NOT_EQUAL(true, "not equals to"),
+
+	// Boolean
+	IS_TRUE(false, "is true"),
+	IS_FALSE(false, "is false"),
+
+	// Number
+	IS_GREATER(true, "is greater than"),
+	IS_GREATER_OR_EQUAL(true, "is greater or equal to"),
+	IS_LESS(true, "is less than"),
+	IS_LESS_OR_EQUAL(true, "is less or equal to"),
+
+	// String
+	STARTS_WITH(true, "starts with"),
+	ENDS_WITH(true, "ends with"),
+	CONTAINS(true, "contains substring");
+
+	private static List<SchematronOperation> operationsOnBooleans = new ArrayList<>(4);
```

Comment:
Replace these lookups with a method for each enum type that lists it's operations:

    enum FeatureType{
        public EnumSet<SchematronOperation> getOperations();
    }